src/profile.ts

```diff
@@ -43,6 +43,19 @@ export class Profiler {
     return minBegin;
   }
 
+  benchmarkOutput(): string {
```

Comment:
Nit: Naming this... `getBenchmarkOutput()` ? Reads better that way. :)

Reply:
I've changed it into a getter: `get benchmarkOutput`